frontend-web/webclient/app/Applications/Card.tsx

```diff
@@ -406,34 +389,41 @@ function lineCountFromType(t: ApplicationCardType): number {
         case ApplicationCardType.EXTRA_TALL:
             return 4;
         case ApplicationCardType.WIDE:
-            return 5;
+            return 4;
         case ApplicationCardType.EXTRA_WIDE:
             return 9;
     }
 }
 
```

Comment:
I know this was originally my function, but this could be replaced by an object:
```
const typeLineCount = {
     [ApplicationCardType.EXTRA_TALL]: 4,
     /**/
}
```